src/Lucid/QueryBuilder/methods.js

```diff
@@ -464,8 +464,13 @@ methods.ids = function (target) {
  * @return {Function}
  */
 methods.pair = function (target) {
+  // console.log(target)
```

Comment:
This the not the right way to do it. Instead we should apply all the global scopes when they exists. Same is done with `fetch` method.

https://github.com/webdevian/adonis-lucid/blob/b043d233bb0d08896abe28e9b894a1443fdadd36/src/Lucid/QueryBuilder/Serializers/Base.js#L37